lib/Watching.js

```diff
@@ -43,7 +43,9 @@ class Watching {
 			this.watchOptions = {};
 		}
 		this.watchOptions.aggregateTimeout =
-			this.watchOptions.aggregateTimeout || 200;
+			this.watchOptions.aggregateTimeout === 0
+				? 0
+				: this.watchOptions.aggregateTimeout || 200;
```

Comment:
`if (this.watchOptions.aggregateTimeout !== undefined) ? this.watchOptions.aggregateTimeout : 200`

Reply:
@evilebottnawi you meant - 

```
this.watchOptions.aggregateTimeout = 
       (this.watchOptions.aggregateTimeout !== undefined ) ? 
                 this.watchOptions.aggregateTimeout : 200 ;
```

Reply:
yes

Reply:
``` js
if (typeof this.watchOptions.aggregateTimeout !== "number") {
  this.watchOptions.aggregateTimeout = 200;
}
```

Reply:
Done :)